### ISSUE TITLE
Issue with multiple instances of EffekseerEmitters

### ISSUE THREAD

**Issue by SebastianKuhlmann** [effekseer/EffekseerForGodot3#13]
There is an issue when having multiple instances of the same particle effect spawned at the same time/shortly after another. As soon as a new particle emiter spawns, the old Effekseer-Emitter (which should continue playing) gets reseted and doesn't finish playing.
I tried to prevent this by activating the property "local to scene" under the "resources" section of the EffekseerEmitter. But that doesn't help. Simply duplicating (in script) the particle file (.efkecf file) / the scene with the EffekseerEmitter atached doesen't work either.
To sum up: I am unable to spawn unique and independant Effekseer-Emiters. There is a video for illustration atached.

https://user-images.githubusercontent.com/82902572/207884344-24fdbbdd-bbac-4da2-a6be-5207d55b327b.mp4

**Comment by ueshita** [comment]
It may be that the instance pool is exhausted.
Can you try increasing the "Instance Max Count" number to see if that solves the issue?
![image](https://user-images.githubusercontent.com/1732153/212237053-30787919-9115-465c-8085-537b416f64fe.png)
Thanks!
